src/plugins/intel_cpu/src/nodes/string_tensor_pack.cpp

```diff
@@ -81,6 +81,10 @@ struct StringTensorPack::StringTensorPackExecute {
     }
 };
 
+bool StringTensorPack::isExecutable() const {
+    return !(isInputTensorAtPortEmpty(0) || isInputTensorAtPortEmpty(1));
+}
+
```

Comment:
It's great that the issue is fixed for the reproducer script, but is it possible to add a unit test to cover such case?

Reply:
Tests have been added.